EIPS/eip-5023.md

```diff
@@ -0,0 +1,144 @@
+
+These requirements have set us to define shareable non-transferable NFTs. These shareable NFTs can be “shared” in the same way digital goods can be shared, at an almost zero technical transaction cost. They are used to instantiate quantified anti-rival value, a medium of sharing. Hence, they work somewhat as money, being a store of value and a unit of account, but instead of being a medium of exchange, they are a medium of sharing.
+
+Typical NFT standards such as ERC-721 and ERC-1155 do not define a sharing modality. Instead ERC standards define user interfaces for typical rival use cases such as token minting and token transactions that the NFT contract implementations should fulfil. The ‘standard’ contract implementations may extend the functionalities beyond the definition of interfaces. The tokens developed in the ATARCA experiments are designed to be token standard compatible at the interface level. However the implementation of token contracts may contain extended functionalities to match the requirements of the experiments such as the requirement of 'shareability'. In reflection to standard token definitions, shareability of a token could be thought of as re-mintability of an existing token. Contracts define re-mintable non-transferable tokens which retain some reference to previous tokens upon and after re-minting.
+
+## Specification
+
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.
+
+```solidity
+///  Note: the ERC-165 identifier for this interface is 0xd763f0cf
+interface IERC5023 is IERC165 {
+
+  /// @dev This emits when a token is shared, reminted and given to another wallet that isn't function caller
+  event Share(address indexed from, address indexed to, uint256 indexed tokenId, uint256 derivedFromtokenId);
+
+  /// @dev Shares, remints an existing token, gives a newly minted token a fresh token id, keeps original token at function callers possession and transfers newly minted token to receiver which should be another address than function caller. 
+  function share(address to, uint256 tokenIdToBeShared, uint256 newTokenId) external;
+
+} 
+```
+
+## Rationale
+
+Current NFT standards define transferable non-fungible tokens. However to be able to create shareable NFTs we see that existing NFT contracts could be extended with an interface which defines the basic principles of sharing, namely the Event of sharing and the function method of sharing. In ATARCA we have chosen to go with shareable non-transferable NFTs which in our use case denote an award or merit for done effort. 
```

Comment:
This section reads like motivation (because it's describing why the EIP as a whole exists.) The Rationale section is meant to explain technical choices within the EIP itself. For example, you might want to describe why `share` takes `newTokenId` as a parameter, and isn't a return value.

Reply:
Updated in the PR.